mocker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Github Repository Link
+GH_REPO=github.com/bakurits/mattermost-plugin-anonymous
+
+# Array of (destination,interface) tuples
```

Comment:
`package` will be more suitable than `destination`

Reply:
Right, package is better name